project.clj

```diff
@@ -36,7 +37,7 @@
                            :jar-exclusions ^:replace []
                            :source-paths ^:replace ["src/clojure" "src/java"]}}
 
-  :plugins [[lein-release "1.0.5"]]
+  :plugins [[puppetlabs/i18n "0.4.3"]]
```

Comment:
I took `lein-release` out b/c it had a dependency conflict with i18n and shouldn't be needed anymore anyway since we're now using versions of leiningen for builds where `lein-release` is built it, AFAIK.